<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Z_POS_v5</string>
  </property>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>통신 초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>연결</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cert">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>인증 (100)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_coupon_num">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_approve">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>승인 (101)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>승인 취소 (102)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_disconnect">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>연결 종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>쿠폰번호</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_browser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>1151</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_exchange_id">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="placeholderText">
    <string>EC0000</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_branch_code">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_branch_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>브랜드 ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>지점 코드</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>지점명</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_coupon_use_amt">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="placeholderText">
    <string>승인금액</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_admit_num">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="placeholderText">
    <string>승인번호</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pos_code">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>포스코드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_net_cancel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>승인 망 취소 (103)</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>20</width>
     <height>291</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_form_clear">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>300</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>폼 지우기</string>
   </property>
  </widget>
  <widget class="QFrame" name="service_type_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>661</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(2, 124, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_tcp">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TCP/IP</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_web">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WEB POS</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupPosMode">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>110</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>POS MODE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_version">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>돋움</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>버전</string>
   </property>
  </widget>
  <zorder>groupPosMode</zorder>
  <zorder>btn_connect</zorder>
  <zorder>btn_cert</zorder>
  <zorder>line_coupon_num</zorder>
  <zorder>btn_approve</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>btn_disconnect</zorder>
  <zorder>label</zorder>
  <zorder>text_browser</zorder>
  <zorder>line_exchange_id</zorder>
  <zorder>line_branch_code</zorder>
  <zorder>line_branch_name</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>btn_clear</zorder>
  <zorder>line_coupon_use_amt</zorder>
  <zorder>line_admit_num</zorder>
  <zorder>btn_reset</zorder>
  <zorder>line_pos_code</zorder>
  <zorder>label_5</zorder>
  <zorder>btn_net_cancel</zorder>
  <zorder>line</zorder>
  <zorder>btn_form_clear</zorder>
  <zorder>service_type_label</zorder>
  <zorder>radio_web</zorder>
  <zorder>radio_tcp</zorder>
  <zorder>line_version</zorder>
  <zorder>label_6</zorder>
 </widget>
 <tabstops>
  <tabstop>btn_reset</tabstop>
  <tabstop>btn_connect</tabstop>
  <tabstop>btn_disconnect</tabstop>
  <tabstop>line_coupon_num</tabstop>
  <tabstop>line_exchange_id</tabstop>
  <tabstop>line_branch_code</tabstop>
  <tabstop>line_branch_name</tabstop>
  <tabstop>line_pos_code</tabstop>
  <tabstop>btn_cert</tabstop>
  <tabstop>line_coupon_use_amt</tabstop>
  <tabstop>btn_approve</tabstop>
  <tabstop>line_admit_num</tabstop>
  <tabstop>btn_cancel</tabstop>
  <tabstop>btn_net_cancel</tabstop>
  <tabstop>text_browser</tabstop>
  <tabstop>btn_form_clear</tabstop>
  <tabstop>btn_clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
